ArduPlane/qautotune.cpp

```diff
@@ -41,8 +41,9 @@ void QAutoTune::get_pilot_desired_rp_yrate_cd(float &des_roll_cd, float &des_pit
 
 void QAutoTune::init_z_limits()
 {
-    plane.quadplane.pos_control->set_max_speed_z(-plane.quadplane.get_pilot_velocity_z_max_dn(), plane.quadplane.pilot_velocity_z_max_up);
-    plane.quadplane.pos_control->set_max_accel_z(plane.quadplane.pilot_accel_z);
+    plane.quadplane.pos_control->set_max_speed_accel_z(-plane.quadplane.get_pilot_velocity_z_max_dn(),
```

Comment:
Randy said no more name changes. The units are in the parameter names so that is probably good enough.
void set_max_speed_accel_xy(float speed_cms, float accel_cmss, float accel_limit_cmss = 0.0f);